stdlib/format.mli

```diff
@@ -1246,6 +1246,21 @@ val pp_print_text : formatter -> string -> unit
   @since 4.02
 *)
 
+val format_text: ('a,'b,'c,'d,'e,'f) format6 -> ('a,'b,'c,'d,'e,'f) format6
+(** [text fmt] replaces spaces and newlines in the format string literal [fmt]
+    with hint breaks or forced newlines:
+- Only ascii spaces ([" " = "\u{20}"]) and linefeeds ["\n" = "\u{A}"] are
+  replaced. In particular, this means that non-breaking spaces (for instance
+  ["\u{A0}"]) are left unchanged.
+- [0<=k<=1] newlines followed by [n] spaces are converted to a [@;<0 (k+n)>]
+  break hint (when [k+n>=1]).
+- Two and more newlines followed by spaces are converted to two forced
+  newlines ["@\n@\n"].
```

Comment:
I find it odd that the function performs newline collapsing on paragraphs separation but not space collapsing. Is there any rationale to this ? 

I think I'd rather have either both collapse or neither.

Reply:
You are right, it is better to be uniform in term of collapsing behavior.